docker/Dockerfile.base

```diff
@@ -2,7 +2,7 @@
 # and scripts/setup_macos.sh, if there's any changes applied to this file,
 # specially regarding the installation of dependencies, apply those same
 # changes to the mentioned files.
-ARG PARENT_IMAGE=ubuntu:16.04
+ARG PARENT_IMAGE=ubuntu:18.04
```

Comment:
there will be several changes between the 18.04 and 16.04 Dockerfiles -- the 18.04 file will likely be shorter (e.g. no need to recompile GLFW from source). Ask Aravind for what he does for 18.04. The xvfb config may or may not change. 

Try to remove as much as you can, since everything makes the Dockerfile bigger and we have probably accumulated things we don't need.

See: https://github.com/openai/gym/blob/0.11.0/test.dockerfile.18.04

Reply:
About GLFW, we still need to recompile it from source because ubuntu 18.04 ships with glfw 3.2.1 which still has the same issue (https://github.com/glfw/glfw/issues/1004) and glfw 3.3 is not available in ubuntu software repos yet